libbeat/cloudid/cloudid.go

```diff
@@ -117,8 +117,8 @@ func extractPortFromName(word string, defaultPort string) (id, port string) {
 	return word, defaultPort
 }
 
-// decodeCloudID decodes the cloud.id into elasticsearch-URL and kibana-URL
-func decodeCloudID(cloudID string) (string, string, error) {
+// DecodeCloudID decodes the cloud.id into elasticsearch-URL and kibana-URL
+func DecodeCloudID(cloudID string) (string, string, error) {
```

Comment:
If we start to expose this function, we should also try to make it more clear what is returned. Either by introducing a new type or naming the return values. I'd prefer a struct like:

```
type CloudIDDetails struct {
  ElasticsearchURL string
  KibanaURL string
}
```